Gemfile.lock

```diff
@@ -168,6 +171,8 @@ GEM
     nokogiri (1.6.7.2)
       mini_portile2 (~> 2.0.0.rc2)
     normalize-rails (3.0.3)
+    parser (2.3.0.7)
```

Comment:
Notice how the `parser` gem has its version reflect the version of ruby. It was a dependency of `i18n-tasks` which meant a bump of it was needed.